spec/Body/StaticLights.vspec

```diff
@@ -0,0 +1,12 @@
+# (C) 2022 BMW Group - All rights reserved.
```

Comment:
In other parts of code BMW is referred to as "Bayerische Motoren Werke Aktiengesellschaft (BMW AG)". Is it intentional to use a new name, if so, shall the old copyright be updated?